rules/docker-dhcp-relay.mk

```diff
@@ -45,7 +45,7 @@ endif
 endif
 
 $(DOCKER_DHCP_RELAY)_CONTAINER_NAME = dhcp_relay
-$(DOCKER_DHCP_RELAY)_CONTAINER_PRIVILEGED = true
+$(DOCKER_DHCP_RELAY)_CONTAINER_PRIVILEGED = false
```

Comment:
Quick question - why is this container different than the others?
In the others we have the run_opt as the flag defining the privileged state of the container.

Reply:
I think it's related to this PR, making dhcprelay a sonic application extension, https://github.com/sonic-net/sonic-buildimage/pull/6531/files, HLD https://github.com/sonic-net/SONiC/blob/master/doc/sonic-application-extension/sonic-application-extention-hld.md